.github/chainguard/self.create-version-bump-pr.create-pr.sts.yaml

```
@@ -0,0 +1,13 @@
+issuer: https://token.actions.githubusercontent.com
+
+subject: repo:DataDog/datadog-aas-extension:ref:refs/heads/main
```

This means we can only run the workflows against `main`, correct? Which means we can't test changes to the workflows themselves. Is it worth doing something like the following (which we have in dd-trace-dotnet)

```suggestion
subject_pattern: repo:DataDog/datadog-aas-extension:.*
```

---

I updated the code to address both comments